presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/MongoQueryRunner.java

```diff
@@ -32,55 +31,53 @@
 import static io.prestosql.testing.TestingSession.testSessionBuilder;
 
 public class MongoQueryRunner
-        extends DistributedQueryRunner
 {
     private static final String TPCH_SCHEMA = "tpch";
 
+    private final DistributedQueryRunner queryRunner;
     private final MongoServer server;
     private final MongoClient client;
 
-    private MongoQueryRunner(Session session, int workers)
-            throws Exception
+    private MongoQueryRunner(Iterable<TpchTable<?>> tables)
```

Comment:
Typically we just provide utility function -- like `io.prestosql.plugin.cassandra.CassandraQueryRunner#createCassandraQueryRunner`